packages/react-server/src/ReactFlightServer.js

```diff
@@ -3354,6 +3354,27 @@ function renderModelDestructive(
           task.debugOwner = element._owner;
           task.debugStack = element._debugStack;
           task.debugTask = element._debugTask;
+          if (
+            element._owner === undefined ||
+            element._debugStack === undefined ||
+            element._debugTask === undefined
+          ) {
+            let key = '';
+            if (element.key !== null) {
+              key = ' key="' + element.key + '"';
+            }
+
+            console.error(
```

Comment:
We usually don't do conditional throws but maybe this here is a valid exception since we'll throw later anyway in dev when accessing `.stack`?

Reply:
We might fix that to not throw though and then it would become unbalanced.